### PR TITLE
Rails Initiate

### PR DESCRIPTION
**Pull request by Prozak8**
https://www.pivotaltracker.com/story/show/160920353

Tasks:
- Setting up the Rails project. 
- Cleaning up the commented text
- Placeholder README

### CODE REVIEW

**Comment by ghost** [comment]
Please review the PT task descriptions in order to match all the content of PR.  
Remove the unnecessary remark in codes, thank you

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.5.1'
+gem 'rails', '~> 5.2.1'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 3.11'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.2'
```
Comment (faraznaeem) [inline Gemfile]: remove coffee script

File: config/application.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.2
+
+    # Settings in config/environments/* take precedence over those specified here.
+    # Application configuration can go into files in config/initializers
+    # -- all .rb files in that directory are automatically loaded after loading
+    # the framework and any gems in your application.
+
+    # Don't generate system test files.
+    config.generators.system_tests = nil
```
Comment (faraznaeem) [inline config/application.rb:21]: You should remove the comments from this file

**Review by faraznaeem — CHANGES_REQUESTED** [review]
The PT story is wrong, it links me to create a Repo not for the scaffolding of the application. 
I would also suggest that you add Haml as the templating engine. If you choose to do that as a separate chore or in this PR I leave up to you guys.

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+gem 'turbolinks', '~> 5'
+gem 'jbuilder', '~> 2.5'
```
Comment (tochman) [inline Gemfile]: Please remove jbuilder

**Comment by faraznaeem** [comment]
And I can see that Shrimen also pointed out the PT story

File: Gemfile [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+gem 'bootsnap', '>= 1.1.0', require: false
+
+group :development, :test do
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```
Comment (tochman) [inline Gemfile]: please remove byebug

File: Gemfile [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+end
+
+group :development do
+  gem 'web-console', '>= 3.3.0'
+  gem 'listen', '>= 3.0.5', '< 3.2'
+  gem 'spring'
+  gem 'spring-watcher-listen', '~> 2.0.0'
+end
+
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```
Comment (tochman) [inline Gemfile]: Please remove this line. We are not making use of this gem

File: README.md [hunk 6]
```diff
@@ -1,2 +1,6 @@
-# cattastic
-For all the the cats out there!
+###Instructions
+Pulling from https://thecatapi.com/ we will create a cat comparison web app!
```
Comment (tochman) [inline README.md:2]: Please include a space between the hashtags and the header for proper markdown syntax

File: README.md [hunk 7]
```diff
@@ -1,2 +1,6 @@
-# cattastic
-For all the the cats out there!
+###Instructions
+Pulling from https://thecatapi.com/ we will create a cat comparison web app! 
+
+
+###Learning objective
```
Comment (tochman) [inline README.md]: Please include a space between the hashtags and the header for proper markdown syntax

**Review by tochman — APPROVED** [review]
Lgmo

**Pull request merged**